dev/make-distribution.sh

```diff
@@ -29,7 +29,7 @@ set -e
 set -x
 
 # Figure out where the Spark framework is installed
-SPARK_HOME="$(cd "`dirname "$0"`"; pwd)"
+SPARK_HOME="$(cd "`dirname "$0"`/.."; pwd)"
```

Comment:
Can we update those tools to look for the file in both? It also really bugs me that this is a top level file ...